AUTHORS.txt

```diff
@@ -330,6 +330,7 @@ Julian Berman
 Julian Gethmann
 Julien Demoor
 Jussi Kukkonen
+Justin Perez
```

Comment:
Don’t manually modify this file; it is automatically generated from commit history.